src/docs/asciidoc/Batch-Jobs.asciidoc

```diff
@@ -509,6 +510,25 @@ xd:> job create partitionedJob --definition "jdbchdfs --columns='id,col1,col2' -
 
 NOTE: When using the partitioning support you can not use the `sql` option. Use `tableName` and `columns` instead.
 
+You can perform incremental imports using this job by defining a column to check against.  Currently the column must be numeric (similar to how the partitionColumn works).  An example of launching a job that performs incremental imports would look like the following:
+
+----
+xd:> job create incrementalImportJob --definition "jdbchdfs --columns='id,col1,col2' --tableName=some_table --checkColumn=sequence" --deploy
+----
+
+If you want to specify the value for the `checkColumn`, you can pass the override value in as a `JobParameter` named `overrideCheckColumnValue` as shown below:
+
+----
+xd:> job launch incrementalImportJob --params {"overrideCheckColumnValue" : 2}
+----
+
+There are two things to keep in mind when using incremental imports with this job:
+
+* When using incremental imports, the `sql` option is not available.  Use `tableName` and `columns` instead.
+* If an import fails, it must be rerun to completion before running the next import.  Without this, inconsistent data may result.
```

Comment:
Since HDFS isn't a transactional resource, should we add a note to check the exported data for completeness before restarting the job? Should we add the restartable option?